### DISCUSSION TITLE
Spurführung immer (hoch) als globale Einstellung (VCA Settings werden nicht gespeichert)

### DISCUSSION THREAD

**Discussion by Shortylaforge** [Mogli12/KeyboardSteer#730] [Ideas]
Ich habe das Problem, das keine Settings gespeichert werden. Speziell die View der Lines (möchte immer an, Hoch eingestellt haben) stellen sich immer wieder zurück. Sogar bei Fz-wechsel innerhalb eines SaveGames.

- Single Player
- Config Datei im mods_settings Folder vorhanden, kein Schreibschutz, aber LEER

**Comment by Shortylaforge** [comment]
version: FS22 1.0 rc5 Brazilian translation by leo32128fs

**Comment by Mogli12** [comment]
Die Settings werden beim Fahrzeug gespeichert.

**Comment by Shortylaforge** [comment]
Dh wenn das Fz gemodded ist, und vlt, sagen wir nicht sauber "programmiert" wurde, kann ich nix speichern? Wie kann ich denn die Spurführung in welcher Konfig Deines Mods auf Default: (IMMER AN [HOCH]) stellen? Das würde mir ja reichen.

**Comment by Mogli12** [comment]
Lade bitte mal die vehicle.xml und die log.txt in Gist oder Pastebin hoch.

**Comment by Shortylaforge** [comment]
Vehicle.xml https://pastebin.com/ujNRbf9e

**Comment by Shortylaforge** [comment]
log.txt https://pastebin.com/3BR2xWcR

**Comment by Mogli12** [comment]
In der vehicle.xml kann ich beim Mod FS22_MbTrack_37fdf88e_fs22planet folgenden Eintrag sehen:
```xml
 <zzzVehicleControlAddon snapDistance="24.000000" snapDraw="4"/>
```
Hier ist `snapDraw="4"` die gespeicherte Einstellung.

**Comment by Shortylaforge** [comment]
Ich prüfe das mal bei dem Truck ob die Settings beibehalten wurden. Wie kann ich denn das Setting IMMER (HOCH) als Default setzen?

**Comment by Mogli12** [comment]
Momentan gibt es keine globalen Settings dazu.
